picarto/dist/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "author": {
+        "name": "TheSlowly",
+        "id": "505071508851523599"
+    },
+    "service": "Picarto",
+    "description": {
+        "en": "Picarto is the world’s leading live streaming platform for creative minds. Come join us and watch artists from all over the world during their creative process!"
+    },
+    "url": "picarto.tv",
+    "version": "1.0",
+    "logo": "https://i.imgur.com/Oljre0e.png",
+    "thumbnail": "https://i.imgur.com/DSNnMk5.png",
+    "color": "#FF0000",
+    "tags": [ "video" ]
+}
```

Comment:
There is a new required key in the metadata called `category` that you are missing, take at look at [this](https://docs.premid.app/en/dev/presence/metadata) for more information.